tests/exchange/test_binance.py

```diff
@@ -9,12 +9,22 @@
 from tests.exchange.test_exchange import ccxt_exceptionhandlers
 
 
-@pytest.mark.parametrize('limitratio,expected', [
-    (None, 220 * 0.99),
-    (0.99, 220 * 0.99),
-    (0.98, 220 * 0.98),
+@pytest.mark.parametrize('limitratio,exchangelimitratio,expected,side', [
+    (None, 1.05, 220 * 0.99, "sell"),
+    (0.99, 1.05, 220 * 0.99, "sell"),
+    (0.98, 1.05, 220 * 0.98, "sell"),
```

Comment:
This is unrealistic ... 
the value you're assigning to `exchangelimitratio` will not change depending on the direction (assuming we plan to allow a bot to go both long and short, and won't hard-limit a running bot to one side only).

I've a comment on the "implementation" part for this as well iirc ...

Reply:
The exchangelimitratio value was `1.05` before I made changes. The new tests are the ones where the side is "buy"

Reply:
`exchangelimitratio` is also no longer a parameter on the latest commit